src/main/java/mekanism/common/integration/IMCHandler.java

```diff
@@ -24,105 +21,55 @@ public void onIMCEvent(List<IMCMessage> messages)
 		{
 			if(msg.isNBTMessage())
 			{
-				try {
-					boolean found = false;
-					boolean delete = false;
-					
-					String message = msg.key;
-					
-					if(message.equals("ShapedMekanismRecipe"))
```

Comment:
Fixed, logging a warning currently when one of the 4 messages is being received.